tests/env.debug.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
Is there a lot of value in a script that just wait for Jahia and start cypress? 
Is it just to make sure we don't forget to source ?

Reply:
It's there as well :) , @tdraier is that script really useful?

Reply:
this script is not harmful and could ease the local test flow

Reply:
I'm fine either way

Reply:
maybe not very useful but does not harm a lot :-)